README.md

```diff
@@ -1152,6 +1152,7 @@ Table of Contents
   * [videoinu](https://videoinu.com) — Create and edit screen recordings and other videos online.
   * [RandomKeygen](https://randomkeygen.com/) - A free mobile-friendly tool offers a variety of randomly generated keys and passwords you can use to secure any application, service or device.
   * [Cronhooks](https://cronhooks.io/) - Schedule one time or recurring webhooks using api and web app. Free plan allows 1 webhook schedule.
+  * [Hook Relay](https://www.hookrelay.dev/) - Add webhook support to your app without the hassles: done-for-you queueing, retries with backoff, and logging.
```

Comment:
Neat, would you mind adding something like `100 deliveries per day, 14-day retention, and 3 hook endpoints`

Reply:
Ah yeah, sorry about that :)